### ISSUE TITLE
Cert auth per frontend

### ISSUE THREAD

**Issue by Yajo** [containous/traefik#1475]
### What version of Traefik are you using (`traefik version`)?

```
Version:      v1.2.3
Codename:     morbier
Go version:   go1.7.5
Built:        2017-04-13_07:21:10PM
OS/Arch:      linux/amd64
```

### What is your environment & configuration (arguments, toml...)?

Docker + labels.

### What did you do?

I wanted to make sure that only certificate-authenticated clients can connect to a certain subpath of some services.

### What did you expect to see?

A way to do this.

### What did you see instead?

Current Traefik's cert auth is implemented by entrypoint, similar to #751. Obeying https://github.com/containous/traefik/issues/751#issuecomment-295473571, here's its counterpart for cert auth. 😊

**Comment by ldez** [comment]
This is a duplicate of #1465 - Feel free to subscribe there for updates.

**Comment by Yajo** [comment]
Nope, this one is for *cert* auth, whereas #1465 is for *basic* auth.

**Comment by chxzqw** [comment]
HI just wondering any updates on this ? I would like to see client auth on per frontend feature. Not sure if it is implemented. Can't find any information anywhere else.

**Comment by enriquecaballero** [comment]
Was switching from Caddy to Traefik only to hit this wall. Unfortunately, this is a non-negotiable feature for me -- some services I expose, others I hide under two-way SSL. Hate that I'm gonna have to either (1) code up an auth server or (2) return to using Caddy.

**Comment by irvinlim** [comment]
I suppose this applies for forward auth as well right? I see only basic auth per frontend is possible instead of per entrypoint, which pushes the load of authenticating every single request through Traefik onto the authentication mechanism for a single entrypoint.

**Comment by tpdownes** [comment]
This appears to be a duplicate of #3575, or vice versa.
